src/reporters/lang/en.js

```diff
@@ -132,6 +132,8 @@ const messages = {
   linkUnregistered: 'Unregistered $0.',
   linkUsing: 'Using linked module for $0.',
 
+  createUnsupportedScope: 'Scoped packages cannot be installed via yarn create for now - read <github issue> to know why.',
```

Comment:
Because all binaries are merged together inside the `.bin` directory, executing a scoped binary might not call the right one. I'll fix this before merging, but I figured it might be better to have a prototype without this feature so we can agree on the feature's most basic behavior.